### DISCUSSION TITLE
Unable to restore wallet/LN channels

### DISCUSSION THREAD

**Discussion by Falci** [lightningnetwork/lnd#6684] [Troubleshooting] [answered]
I have an umbrel install and I failed to update, so I start a fresh new install with the latest version.
In this newer version, Umbrel don't have the feature to import existing seedphrase/channels.backup.

I can ssh into the server and run commands there. LND runs in a docker so I created some alias so I can call `lncli`.

When I try to `lncli create`, it asks:
* a new password (and confirm);
* if I have a previous seed, and the seed;
* a password used before. (I'm not sure if I had used password before. If I did, I have it stored. It could also be something from umbrel ?!)
* an optional address a look-ahead, which I used the default 2500;

After that, cli returns the seed, tells me to write down and fails with this message:
```
[lncli] rpc error: code = Unknown desc = wallet already unlocked, WalletUnlocker service is no longer available
```

Not sure how to proceed from here.

**Comment by positiveblue** [comment, marked as answer]
Hello @Falci,

For what I understand installing a new umbrel version allows you to use the previously backup 24 words seed. Didn't umbrel ask for it during the set up of the new version? That should already help you recovering the on chain funds. For the channels, do you still have the `channel.backup` file from your old installation? if so you can use `lncli restorechanbackup --multi_file ~/path/to/channel.backup` 

However, remember to do that AFTER importing the right on chain wallet.

What LND version where you using before? it is not advice to run recoveries using a new binary with old data. It would be better if you were able to 
- run the same lnd version as before 
- recover funds
- migrate to new version



DarthCoin has a good post on this https://community.getumbrel.com/t/how-to-restore-your-node-wallet/3602

**Reply by Falci** [reply]
Hi @positiveblue,

Umbrel don't have a restore option in the current UI. However they just [replied with a temporary solution](https://github.com/getumbrel/umbrel/issues/1461#issuecomment-1171479992)

I managed to recover the on-chain before. Didn't realize the `restorechanbackup` needs to be done after onchain wallet finished rescanning. Thanks for that.

I don't know which lnd version I were running before. Currently, it's on `0.15.0-beta`. I just run the command and seems to work. I can see all my channels closing.

I checked DarthCoin's post but those options didn't work for different reasons (no import option, no `.db` files, and Blixt failed)

Thank you very much for the help!
